src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -533,10 +533,10 @@ BookStep<TIn, TOut, TDerived>::forEachOffer (
                 // Issuer requires authorization.  See if offer owner has that.
                 auto const& ownerID = offer.owner();
                 auto const authFlag =
-                    ownerID > issuerID ? lsfHighAuth : lsfLowAuth;
+                    ownerID > issuerID ? lsfLowAuth : lsfHighAuth;
```

Comment:
We're checking if auth is set on the issuer side of the trust line. For these `high/low` trust line flags, I like writing the conditional so the side we're checking is first. I.e.: `issuerID > ownerID ? lsfHighAuth : lsfLowAuth` (nit for sure, but if you don't have a preference I do prefer `issuerId > ownerID`)